bio/reference/ensembl-sequence/wrapper.py

```
@@ -26,6 +27,11 @@
     build=build, release=release
 )
 
+if path.splitetxt(snakemake.output[0])[1] == ".gz":
+    decompress = ""
+else:
+    decompress = " | gzip -d"
```

Ruff (0.8.2)

Use ternary operator `decompress = "" if path.splitetxt(snakemake.output[0])[1] == ".gz" else " | gzip -d"` instead of `if`-`else`-block

Replace `if`-`else`-block with `decompress = "" if path.splitetxt(snakemake.output[0])[1] == ".gz" else " | gzip -d"`

(SIM108)